app/jinja_filters.py

```diff
@@ -9,26 +9,51 @@
 
 from jinja2 import Markup, escape, evalcontextfilter
 
-from babel import units
+from babel import units, numbers
 
 blueprint = flask.Blueprint('filters', __name__)
 
 
+DEFAULT_LOCALE = 'en_GB'
+
+
+@blueprint.app_template_filter()
+def format_currency_for_input(value=None, decimal_places=0, locale=DEFAULT_LOCALE):
+    formatted_number = ''
+    if value:
+        if decimal_places is None or decimal_places == 0:
+            formatted_number = numbers.format_number(value, locale=locale)
+        else:
+            formatted_number = format_currency(value, locale=locale).replace(get_currency_symbol(locale=locale), '')
+
+    return formatted_number
```

Comment:
It might be cleaner and more consistent with the `format_number` method to remove the `formatted_number` variable and return as soon as we have a formatted number
```
return numbers.format_number(value, locale=locale)
return format_currency(value, locale=locale).replace(get_currency_symbol(locale=locale), '')
return ''
```